examples/tests/sleeptenmin.py

```diff
@@ -29,4 +29,4 @@ def test(self):
             if method == 'builtin':
                 time.sleep(length)
             elif method == 'shell':
-                subprocess.run(["sleep", length])
+                subprocess.run(["sleep", length], check=True)
```

Comment:
This seems to not be related to this commit.  Also, it did not influence the bandits results at all.  Maybe did you mean `shell=True` as an standalone change because the method name is "shell"?

Reply:
This was a late fix detected by pylint (W1510) that went to the wront commit, I moved it now to the proper one.
Answering your question, I really meant `check=True`, it's required to make clear the error behaviour. This way it'll raise an exception when the return of the command is non-zero.

Reply:
OK, a quick note about the why of the `check=True` change on the commit message would be helpful.  Also, there's a typo in the commit message (deprecrated).